example/config.yml

```diff
@@ -15,6 +15,7 @@ collections: # A list of collections the CMS should be able to edit
       - {label: "Publish Date", name: "date", widget: "datetime", format: "YYYY-MM-DD hh:mma"}
       - {label: "Cover Image", name: "image", widget: "image", required: false, tagname: ""}
       - {label: "Body", name: "body", widget: "markdown"}
+      - {label: "Some Field", name: "somefield", widget: "string", required: false} # Testing empty field
```

Comment:
We've been using the example case as a testbed more often, and I think it's an anti-pattern overall - let's remove the example changes and just implement the fix for now.